kuwala/core/backend/app/resources/export_catalog/file/download_as_csv.json

```diff
@@ -0,0 +1,21 @@
+{
+  "id": "download_as_csv",
+  "category": "file",
+  "name": "Download as CSV",
+  "icon": "file-csv",
+  "description": "With this export, you can export the result of a block into a CSV File",
```

Comment:
Please stick to the wording of the designs: "With this export block, you can download the result of a transformation or data block as a CSV file."